### PR TITLE
In 140105657 add apparent age

### PR DESCRIPTION
**Pull request by ThomasOSI**
### Pivotal Story

- [**HOTLINE: ** Create Approximate Age field for a Person #140105657](https://www.pivotaltracker.com/story/show/140105657)

### Purpose
Add approximate age (with units)

### Background
Users may not know an exact birthdate

### Notes for Reviewer
Not to be merged until corresponding API has been updated

### CODE REVIEW

File: app/assets/stylesheets/form.scss [hunk 1]
```diff
@@ -144,6 +144,15 @@ input[type='radio']:disabled + label::before {
...
 
+.input-no-header {
+  margin-top: 56px;
+}
+
+.text-between-inputs {
+  -webkit-margin-before: 65px;
+  text-align: center;
+}
+
```
Comment (aaronOSI) [inline app/assets/stylesheets/form.scss]: Are we sure these styles are not already handled by wood duck classes?

File: app/javascript/screenings/ParticipantCardView.jsx [hunk 2]
```diff
@@ -44,6 +45,13 @@ export default class ParticipantCardView extends React.Component {
     this.props.onChange(this.props.participant.get('id'), participant)
   }
 
+  onDobBlur(value) {
+    if (value) {
+      const participant = this.props.participant.setIn(['approximate_age'], '').setIn(['approximate_age_units'], '')
```
Comment (aaronOSI) [inline app/javascript/screenings/ParticipantCardView.jsx]: Any sign of async-related bugs from chaining setState like this?

File: app/assets/stylesheets/form.scss (hunk 1 above)
Comment (ThomasOSI) [reply app/assets/stylesheets/form.scss]: Yes.  I checked with Pierre before adding them.

File: app/javascript/screenings/ParticipantCardView.jsx (hunk 2 above)
Comment (ThomasOSI) [reply app/javascript/screenings/ParticipantCardView.jsx]: Not that I've seen.

[the remark this answers is not in the document]
Comment (ThomasOSI) [reply app/javascript/screenings/ParticipantCardView.jsx]: Thanks for info on set vs. setIn. I considered tying it to an action but decided to do that as a refactor if there was time.

**Pull request merged**